src/modules/accounting/accounting.py

```diff
@@ -28,4 +32,25 @@ def execute_module(self, blockstamp: BlockStamp):
             self.build_report(report_blockstamp)
 
     def build_report(self, blockstamp: BlockStamp):
-        pass
+        member_info = self.get_member_info(blockstamp)
+        frame_config = self.get_frame_config(blockstamp)
+        chain_config = self.get_chain_config(blockstamp)
+
+        all_validators, lido_keys, lido_validators = self.w3.lido_validators.get_lido_validators_with_others(blockstamp)
+        logger.info({"msg": f"Validators - all: {len(all_validators)} lido: {len(lido_validators)}"})
+
+        cdtp = CommonDataToProcess(
```

Comment:
We are caching almost all requests to nodes, so there is no need to store them like this.
We can just call request data again.
Part of this won't be used in Accounting or in Ejector, but supporting such big structure will take some time and reduce readability

Reply:
Agree. Will be reforged